ContactsApp/ContactsApp/XIB/CustomCell.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CustomCell.swift
+//  ContactsApp
+//
+//  Created by Apple Macbook Pro 13 on 13.08.22.
+//
+
+import UIKit
+
+class CustomCell: UIView {
+
+    @IBOutlet weak var infoLabel: UILabel!
+
+    override init(frame: CGRect){
+        super.init(frame: frame)
+        xibSetup()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        xibSetup()
+    }
+}
+
+private extension CustomCell {
+    func xibSetup() {
+        guard let view = CustomCell.fromNib(withOwner: self) else {
+            return
+        }
+        
+        addSubview(view)
+        view.frame = self.bounds
+        view.autoresizingMask = [.flexibleHeight, .flexibleWidth]
+        view.backgroundColor = .clear
+    }
+    
+}
+
+extension CustomCell {
```

Comment:
make this extension for UIView, not for `CustomCell`. You can reuse this extension `fromNib func` for other cells